src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -226,9 +226,17 @@ public static ReminderDescription parseReminderDescription(String description) t
     public static ReminderDate parseReminderDate(String reminderDate) throws ParseException {
         requireNonNull(reminderDate);
         String trimmedDate = reminderDate.trim();
+
         if (!ReminderDate.isValidDate(trimmedDate)) {
             throw new ParseException(DocumentedDate.MESSAGE_CONSTRAINTS);
         }
-        return ReminderDate.parse(trimmedDate);
+
+        ReminderDate parsedDate = ReminderDate.parse(reminderDate);
+
+        if (parsedDate.getDaysPassed() > 0) {
```

Comment:
would this throw and error if the user wants to set a reminder of the day itself ?

Reply:
also for this check, i think its possible to put it in reminderdate class. it should be more slap. you can look at how isValidContactedDate implemented in RecentDate class

Reply:
Setting reminders on the day itself is okay, as for the check for is valid contacted date, we refactored it to be in parserUtil also